### ISSUE TITLE
Generate XML based on XSD

### ISSUE THREAD

**Issue by 1897** [NeTEx-CEN/NeTEx#89]
Hello,
Whenever i'm trying to generate a xml based on xsd i'm getting following: 

Error: Error generating classes for schema ' '.
  - The element 'http://www.opengis.net/gml/3.2:pos' is missing.

I'm using the **Developer Command Promt for VS 2019**

My goal is to generate the NeTEx class.

How can i fix above issue?

**Comment by skinkie** [comment]
If you would even succeed with this please let us know.

But the actual problem seems to be that MS-XML is not able to handle XSDs with multiple includes.

**Comment by 1897** [comment]
> If you would even succeed with this please let us know.
> 
> But the actual problem seems to be that MS-XML is not able to handle XSDs with multiple includes.

i was hopping you will help me :D

are there any other ways i can achieve it? 
do not you have a complete xml netex file? i have generated class based on a xml but not sure how complete is it

**Comment by skinkie** [comment]
> i was hopping you will help me :D
I have actually asked someone at MSFT some months ago to see if they could help with this on a technical perspective.

> are there any other ways i can achieve it?

XMLSpy and/or OxygenXML were able to generate interfaces for XML processing in C# as far as I understood.

> do not you have a complete xml netex file? i have generated class based on a xml but not sure how complete is it

Sure, but keep in mind, these are NeTEx profiles:
http://data.ndovloket.nl/netex/ (Dutch Profile)
https://github.com/entur/profile-norway-examples (Nordic Profile)
https://data.public.lu/en/datasets/horaires-et-arrets-des-transport-publics/ (EPIP profile)

**Comment by Aurige** [comment]
@1897 what root element are you trying to generate ?

Also, about Profiles, the French ones are here http://www.normes-donnees-tc.org/format-dechange/donnees-theoriques/netex/ (files are http://www.normes-donnees-tc.org/wp-content/uploads/2019/11/NF_Profil-NeTEx-%C3%A9l%C3%A9ments-communsF-v2.1.pdf http://www.normes-donnees-tc.org/wp-content/uploads/2019/11/NF_Profil-NeTEx-pour-les-arr%C3%AAtsF-v2.1.pdf http://www.normes-donnees-tc.org/wp-content/uploads/2019/11/NF_Profil-NeTEx-pour-les-R%C3%A9seauxF-v2.1.pdf http://www.normes-donnees-tc.org/wp-content/uploads/2019/11/NF_Profil-NeTEx-pour-les-HorairesF-v2.1.pdf and for accessibility http://www.normes-donnees-tc.org/wp-content/uploads/2019/12/NF_Profil-NeTEx-pour-laccessibilit%C3%A9F-v1.5.pdf) and UK also have a Tarif Profile available.

**Comment by 1897** [comment]
> @1897 what root element are you trying to generate ?
> 
> Also, about Profiles, the French ones are here http://www.normes-donnees-tc.org/format-dechange/donnees-theoriques/netex/ (files are http://www.normes-donnees-tc.org/wp-content/uploads/2019/11/NF_Profil-NeTEx-%C3%A9l%C3%A9ments-communsF-v2.1.pdf http://www.normes-donnees-tc.org/wp-content/uploads/2019/11/NF_Profil-NeTEx-pour-les-arr%C3%AAtsF-v2.1.pdf http://www.normes-donnees-tc.org/wp-content/uploads/2019/11/NF_Profil-NeTEx-pour-les-R%C3%A9seauxF-v2.1.pdf http://www.normes-donnees-tc.org/wp-content/uploads/2019/11/NF_Profil-NeTEx-pour-les-HorairesF-v2.1.pdf and for accessibility http://www.normes-donnees-tc.org/wp-content/uploads/2019/12/NF_Profil-NeTEx-pour-laccessibilit%C3%A9F-v1.5.pdf) and UK also have a Tarif Profile available.


thanks for your answer!
my primary goal is to convert PPT file to NeTex.
as I understood, a direct conversion is not possible, so it is planned to have an additional file that that will enrich the PPT so we can achieve NeTex. The problem is I have no idea how can things be **mapped** between those 2 standards, what can be used for conversion, what is useless and what is missing in PPT. I know that some parts(fares) of Bison were already migrated to NeTEx, so it can be, probably, easily migrated, what about the rest? 

hopefully you can give me some advises, thanks!

**Comment by skinkie** [comment]
PPT = Dutch fares?

**Comment by 1897** [comment]
> PPT = Dutch fares?

yes, i think it is also called Bison, i need to convert somehow, the entire doc, not only the FareFrame

i should take max all possible data from PPT file and to created based on it a Netex one

**Comment by skinkie** [comment]
PPT = NeTEx (but an older version)
So what do you want to convert it to?

**Comment by 1897** [comment]
to the new version, probably

**Comment by skinkie** [comment]
@1897 can you send me an email? stefan at opengeo dot nl

**Comment by ue71603** [comment]
@skinkie  Is there anything open from this? to be documented?

**Comment by skinkie** [comment]
For C# my hopes are not high. I think a Wikipage may help a lot of people. In addition, if we can have one full fledged example for each profile, than people insisting on using C# may generate a model of of it.
